Example/Alps/AppDelegate.swift

```diff
@@ -14,26 +14,53 @@ import AlpsSDK
 class AppDelegate: UIResponder, UIApplicationDelegate {
     
     var window: UIWindow?
+    
+    var matchDelegate: MatchDelegate! = nil
+    var alpsManager: AlpsManager! = nil
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
-        let alpsManager = AlpsManager(apiKey: "eyJ0eXAiOiJKV1QiLCJhbGciOiJFUzI1NiJ9.eyJpc3MiOiJhbHBzIiwic3ViIjoiNWQxYTM2NjgtZjZmYy00Y2MyLWJlYTgtYWNiMzM1M2MyY2U3IiwiYXVkIjpbIlB1YmxpYyJdLCJuYmYiOjE1MDk5ODQyMTQsImlhdCI6MTUwOTk4NDIxNCwianRpIjoiMSJ9.1JxzbbB91Jq2HvxYWjazrKeRXCEJxcc_XU2Tk1j0xNARwwGlS-JWAto8wHH_P6ASQ-7VzIvYv1wUoV8XAG-LVQ", baseURL: "http://146.148.15.57/v5")
         
-        alpsManager.createMainDevice { _ in
+        alpsManager = AlpsManager(
+            apiKey: "eyJ0eXAiOiJKV1QiLCJhbGciOiJFUzI1NiJ9.eyJpc3MiOiJhbHBzIiwic3ViIjoiNWQxYTM2NjgtZjZmYy00Y2MyLWJlYTgtYWNiMzM1M2MyY2U3IiwiYXVkIjpbIlB1YmxpYyJdLCJuYmYiOjE1MTAzMDMwNjIsImlhdCI6MTUxMDMwMzA2MiwianRpIjoiMiJ9.TmFQdEmkDT6RoI8inxb3zb9Hd8hHJ_0PeCsR1FbUp0jrRaFuBA5J1x7mCSkGYlKFmDmr1GjPhEshgG6qNgg1gQ",
+            baseURL: "http://146.148.15.57/v5"
+        )
+        
+        alpsManager.createMainDevice { result in
+            guard case .success(let mainDevice) = result else { return }
+            // Create New Publication
             let publication = Publication(topic: "Test Topic", range: 20, duration: 100, properties:  ["test": "true"])
-            alpsManager.createPublication(publication: publication, completion: { _ in
-                let subscription = Subscription(topic: "Test Topic", range: 20, duration: 100, selector: "test = 'true'")
-                alpsManager.createSubscription(subscription: subscription, completion: { _ in
-                    print("We're all good! 🏔")
-                })
+            self.alpsManager.createPublication(publication: publication, completion: { result in
+                print("Publication was created \(publication)")
             })
+            
+            // Create New Subscription
+            let subscription = Subscription(topic: "Test Topic", range: 20, duration: 100, selector: "test = 'true'")
+            self.alpsManager.createSubscription(subscription: subscription, completion: { _ in
+                print("Subscription was created \(subscription)")
+            })
+            
+            // Start Monitoring
+            self.matchDelegate = MatchDelegate { matches, _ in
+                print("You've got a match!\n\(matches)")
+                UIApplication.shared.registerForRemoteNotifications()
```

Comment:
Why do you register for Remote Notifications here ??

Reply:
Good catch. It's something that I forgot to remove when testing APNS with Łukasz.

If you don't mind - I will get back to this example in next PR.

Reply:
No problem ! I saw that you comment the registerForPushNotifications() in `RemoteNotificationManager` class. 
Did you succeed in getting the device token ?